src/test/java/com/qaprosoft/carina/demo/WebMultipleBrowserTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2013-2018 QAPROSOFT (http://qaprosoft.com/).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.qaprosoft.carina.demo;
+
+import com.qaprosoft.carina.core.foundation.AbstractTest;
+import com.qaprosoft.carina.core.foundation.utils.ownership.MethodOwner;
+import com.qaprosoft.carina.core.foundation.webdriver.Screenshot;
+import com.qaprosoft.carina.core.foundation.webdriver.core.capability.impl.desktop.ChromeCapabilities;
+import com.qaprosoft.carina.core.foundation.webdriver.core.capability.impl.desktop.FirefoxCapabilities;
+import com.qaprosoft.carina.demo.gui.components.NewsItem;
+import com.qaprosoft.carina.demo.gui.pages.BrandModelsPage;
+import com.qaprosoft.carina.demo.gui.pages.HomePage;
+import com.qaprosoft.carina.demo.gui.pages.ModelInfoPage;
+import com.qaprosoft.carina.demo.gui.pages.NewsPage;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testng.Assert;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+/**
+ * This sample shows how initialize multiple drivers and run the tests on different browsers.
+ *
+ * @author qpsdemo
+ */
+public class WebMultipleBrowserTest extends AbstractTest {
+
+    WebDriver firefoxDriver = null;
+    DesiredCapabilities ffCaps = new FirefoxCapabilities().getCapability("Firefox Test");
+    String ffHost = "http://localhost:4444/wd/hub";
+
+    WebDriver chromeDriver = null;
+    DesiredCapabilities chromeCaps = new ChromeCapabilities().getCapability("Chrome Test");
+    String chromeHost = "http://localhost:4545/wd/hub";
+
+    @BeforeSuite
+    public void initializeDrivers() {
+        firefoxDriver = getDriver("firefox", ffCaps, ffHost);
```

Comment:
remove custom selenium urls and reuse getDriver("name", caps) method. default one in _config.properties should serve all requests